contrib/ansible/vagrant/provision.sh

```diff
@@ -0,0 +1,9 @@
+echo "hello, here is  a sample provisioning script that demonstrates everything works"
+ls /vagrant
+### Required for ansible??
+### Otherwise 'no yum module installed'...
+sudo yum install -y yum # needed for the ansible python stuff
+sudo yum install -y docker
+sudo yum install -y openssl # needed for making certs
```

Comment:
This probably belongs in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/contrib/ansible/roles/kubernetes/tasks/gen_certs.yml